.github/workflows/cd-teardown.yml

```diff
@@ -9,21 +9,21 @@ jobs:
     runs-on: ubuntu-20.04
     if: github.event.ref_type == 'branch'
     steps:
-    - name: Prepare for unpublication from npm
-      uses: actions/setup-node@v2.5.1
-      with:
-        node-version: '16.x'
-        registry-url: 'https://registry.npmjs.org'
-    - name: Determine npm tag
-      # Remove non-alphanumeric characters
-      # See https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
-      run: echo "TAG_SLUG=$(echo "${{ github.event.ref }}" | tr -cd '[:alnum:]-')" >> $GITHUB_ENV
-    - name: Remove npm tag for the deleted branch
-      run: |
-        # Unfortunately GitHub Actions does not currently let us do something like
-        #     if: secrets.NPM_TOKEN != ''
-        # so simply skip the command if the env var is not set:
-        if [ -n $NODE_AUTH_TOKEN ]; then npm dist-tag rm @inrupt/solid-client-access-grants $TAG_SLUG; fi
-      env:
-        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-    - run: echo "Package tag [$TAG_SLUG] unpublished."
+      - name: Prepare for unpublication from npm
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      - name: Determine npm tag
+        # Remove non-alphanumeric characters
+        # See https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
+        run: echo "TAG_SLUG=$(echo "${{ github.event.ref }}" | tr -cd '[:alnum:]-')" >> $GITHUB_ENV
+      - name: Remove npm tag for the deleted branch
+        run: |
+          # Unfortunately GitHub Actions does not currently let us do something like
+          #     if: secrets.NPM_TOKEN != ''
+          # so simply skip the command if the env var is not set:
+          if [ -n $NODE_AUTH_TOKEN ]; then npm dist-tag rm @inrupt/solid-client-access-grants $TAG_SLUG; fi
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Generally it's a good idea to keep the secret name and env variable name aligned, as this isn't actually the `NODE_AUTH_TOKEN` but the `NPM_TOKEN` (node doesn't have auth tokens) — the only reason I mention this is because I've seen more complex setups where this becomes an issue.

Reply:
Actually `NODE_AUTH_TOKEN` is the environment variable used by the `setup-node` GH action for publication top NPM: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#publish-to-npmjs-and-gpr-with-npm